src/admin/messages/pl/cmsadmin.php

```diff
@@ -1,134 +1,136 @@
 <?php
```

Comment:
In this file, I replaced `"message_id" => "translation"` to `'message_id' => 'translation'`